pymleda/pymleda.py

```diff
@@ -16,7 +16,9 @@ class SupervisedData:
         Sequence of target names (y) to be used as dependent variables or labels
 
     **kawrgs: 
-        Additional parameters to pass to sklearn's train_test_split(). For more information see hyperlink: `sklearn's function documentation <https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html/>`_.
+        Additional parameters to pass to sklearn's train_test_split(). 
+        In the absence of additional parameters for the default paramteres of train_test_split() are used including test size  = 0.25
```

Comment:
Just a small typo. It should be "parameters" instead of "paramteres".

Reply:
Thank you - I always make that mistake! Fixed.